src/index.html

```diff
@@ -10,8 +10,137 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="field-group">
+        <legend>Personal information</legend>
+        <label for="surname">Surname:
```

Comment:
Let's fix the formatting. Please, do this in the rest of your file

```suggestion
        <label for="surname">
          Surname:
```